.github/workflows/busola-build.yml

```diff
@@ -30,8 +30,11 @@ on:
       - "src/**"
       - "package.json"
       - "Makefile"
-      - "Dockerfile"
-      - "nginx/nginx.conf"
+      - "Dockerfile*"
+      - "nginx/**"
+      - '.github/workflows/pull-kyma-intergation-tests.yml'
+      - 'resources/**'
+      - 'tests/integration/**'
```

Comment:
this change is due to the `kyma-integration-dev` which requires this build to run the tests, so paths should be the same as in test workflow.